dist/stat_data.py

```diff
@@ -235,6 +235,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_maximum_page_size', 'maximum page size at eviction', 'no_clear,no_scale,size'),
     CacheStat('cache_eviction_pages_queued', 'pages queued for eviction'),
```

Comment:
Ideally, we will call this statistic as "pages considered for LRU sorting" or something similar and the new statistic as "pages queued for eviction". But the changes to this statistic might add to the confusion with FTDC within releases and we will have to educate a lot of people around it. @agorrod , what do you think? Any suggestions for naming the new statistic that is really "pages queued".

Reply:
I agree that ideally we'd change the name of this statistic, but also that the fallout is non-trivial. I'm inclined to leave as-is, I think the new stat will sort in a way that makes it clear enough (at least for experts).